docs/Introduction/architecture-overview/architecture-decentralized-model.md

```diff
@@ -11,23 +11,23 @@ metadata:
   image: 
     0: "/files/OpenGraph_V3.png"
 ---
-This page describes how data aggregation is applied to produce Chainlink Price Feeds.
+This page describes how data aggregation is applied to produce Chainlink Data Feeds.
 
 # Price Aggregation
 
-Each price feed is updated by multiple, independent Chainlink oracle operators. Aggregation is handled on-chain by <a href="https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.6/FluxAggregator.sol" target="_blank">`FluxAggregator`</a>.
+Each data feed is updated by multiple, independent Chainlink oracle operators. Aggregation is handled on-chain by <a href="https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.6/FluxAggregator.sol" target="_blank">`FluxAggregator`</a>.
 
 ![Chainlink Feeds List](/images/contract-devs/price-aggr.png)
 
 ## Shared Data Resource
 
-Each price feed is built and funded by the community of users who rely on accurate, up-to-date price data in their smart contracts. As more users rely on and contribute to a price feed, the quality of the price feed improves. For this reason, each price feed has its own properties depending on the needs of its community of users.
+Each data feed is built and funded by the community of users who rely on accurate, up-to-date price data in their smart contracts. As more users rely on and contribute to a data feed, the quality of the data feed improves. For this reason, each data feed has its own properties depending on the needs of its community of users.
 
 ## Decentralized Oracle Network
 
-Each price feed is updated by a decentralized oracle network. Each oracle operator is rewarded for publishing price data. The number of oracles contributing to each feed varies. For example, in the ETH/USD Price Feed, there are 21 oracles.
+Each data feed is updated by a decentralized oracle network. Each oracle operator is rewarded for publishing price data. The number of oracles contributing to each feed varies. For example, in the ETH/USD Data Feed, there are 21 oracles.
 
-In order for an update to take place, the price feed contract must receive responses from a minimum number of oracles. For example, 14 / 21 oracles. Otherwise, the latest answer will not be updated.
+In order for an update to take place, the data feed contract must receive responses from a minimum number of oracles. For example, 14 / 21 oracles. Otherwise, the latest answer will not be updated.
 
 Each oracle in the set publishes answers to the latest price of an asset during an aggregation round. The answers are validated and aggregated by a smart contract, which forms the feed's latest and trusted answer. Developers wishing to use an asset's latest and trusted answer can do so easily by following the [Get the Latest Price](../get-the-latest-price/) page.
 
```

Comment:
There are a few more references to price here that need to be generalized to discuss data.

"Whichever condition is met first will trigger a price update" > "Whichever condition is met first will trigger an update to the data."

"A new aggregation round starts when a node identifies that the off-chain price deviates, by more than the deviation threshold, of the on-chain price. Individual nodes monitor one or more data providers for each feed." > "A new aggregation round starts when a node identifies that the off-chain values deviate by more than the defined deviation threshold from the on-chain value. Individual nodes monitor one or more data providers for each feed."